sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRegistryClientBuilder.java

```diff
@@ -151,40 +206,51 @@ public ContainerRegistryClientBuilder retryPolicy(RetryPolicy retryPolicy) {
 
 
     /**
-     * Sets the service version that will be targeted by the client.
-     * If the version is not set the latest version is used by default.
+     * Sets the {@link ContainerRegistryServiceVersion} that is used when making API requests.
+     * <p>
+     * If a service version is not provided, the service version that will be used will be the latest known service
+     * version based on the version of the client library being used. If no service version is specified, updating to a
+     * newer version the client library will have the result of potentially moving to a newer service version.
      *
-     * @param version the service version to target.
-     * @return the ContainerRegistryBuilder.
+     * @param version {@link ContainerRegistryServiceVersion} of the service to be used when making requests.
+     * @return The updated {@link ContainerRegistryClientBuilder} object.
      */
     public ContainerRegistryClientBuilder serviceVersion(ContainerRegistryServiceVersion version) {
         this.version = version;
         return this;
     }
 
     /**
-     * Adds a custom Http httpPipeline policy.
+     * Adds a policy to the set of existing policies.
      *
-     * @throws NullPointerException if custom policy is null.
-     * @param customPolicy The custom Http httpPipeline policy to add.
-     * @return the ContainerRegistryBuilder.
+     * @param policy The policy for service requests.
+     * @return The updated ConfigurationClientBuilder object.
+     * @throws NullPointerException If {@code policy} is null.
      */
-    public ContainerRegistryClientBuilder addPolicy(HttpPipelinePolicy customPolicy) {
-        Objects.requireNonNull(customPolicy, "'customPolicy' cannot be null.");
+    public ContainerRegistryClientBuilder addPolicy(HttpPipelinePolicy policy) {
+        Objects.requireNonNull(policy, "'customPolicy' cannot be null.");
```

Comment:
Use the same param name in the error message.
```suggestion
        Objects.requireNonNull(policy, "'policy' cannot be null.");
```